stripe/src/main/java/com/stripe/android/model/Source.java

```diff
@@ -298,6 +309,10 @@ public void setUsage(@Usage String usage) {
         putStripeJsonModelMapIfNotNull(hashMap, FIELD_RECEIVER, mReceiver);
         putStripeJsonModelMapIfNotNull(hashMap, FIELD_REDIRECT, mRedirect);
 
+        if (mType != null) {
```

Comment:
I was thinking about adding type to the constructor and having it be nonnull, so that we never have to check in this statement. I was worried that was changing more code than was otherwise necessary for the diff. Happy to put it in though.